src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -666,7 +666,7 @@ def load_arguments(self, _):
 
         with self.argument_context(scope, arg_group='Authentication') as c:
             c.argument('generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing. The keys will be stored in the ~/.ssh directory')
-            c.argument('admin_username', help='Username for the VM.', default=get_default_admin_username())
+            c.argument('admin_username', help='Username for the VM. Default value is current username of OS. If the default value is invalid, the username will be set AzureUser.', default=get_default_admin_username())
```

Comment:
```suggestion
            c.argument('admin_username', help='Username for the VM. Default value is current username of OS. If the default value is system reserved, the username will be set AzureUser. To get full list of system reserved username, pls refer to xxx', default=get_default_admin_username())
```

Reply:
the username will be ~~set~~ AzureUser
or
the username will be set **to** AzureUser